draft-jholland-mops-taxonomy.md

```diff
@@ -96,50 +106,48 @@ technologies address or avoid these existing observed problems.
 
 ###Video Bitrates
 
-Video bit-rate selection depends on many variables.  Different
+Video bitrate selection depends on many variables.  Different
 providers give different guidelines, but an equation that
 approximately matches the bandwidth requirement estimates
 from several video providers is given in {{MSOD}}:
 
 ~~~
-Kbps = (HEIGHT * WIDTH * FRAME_RATE) / (7 * 1024)
+Kbps = (HEIGHT * WIDTH * FRAME_RATE) / (15 * 1024)
```

Comment:
I felt like the resulting numbers were quite high (and scary to be frank). No provider really uses that high bitrates for consumer delivery (they do use for contribution, though). Maybe we can add a sentence saying the typical numbers are about the half of the numbers given by the formula.

Reply:
Or maybe the text could be refactored to include both, instead of picking one?  I'd like to encourage people to plan for the higher end, to the extent someone might end up using these numbers as a guideline.

Reply:
That works.